package.json

```diff
@@ -51,14 +51,15 @@
   },
   "scripts": {
     "postinstall": "yarn --cwd ./examples",
-    "prebuild": "rimraf dist && npm run typegen",
-    "build": "rollup -c && yarn copy && rimraf dist/__tests__",
+    "build": "rimraf dist && npm run typegen && rollup -c && yarn copy && rimraf dist/__tests__ && npm run replace-reconciler-imports",
     "prepare": "npm run build",
     "eslint": "eslint --fix src/**/*.{ts,tsx}",
     "dev": "yarn --cwd ./examples start",
-    "test": "jest",
-    "typecheck": "tsc --noEmit --emitDeclarationOnly false --strict --jsx react",
-    "typegen": "tsc && mv dist/src/* dist && rm -rf dist/src || true",
+    "tests": "jest",
```

Comment:
Duplicated test and tests? Can this be more clear?

Reply:
`test` (standard name) runs all linting, typechecking and tests. `tests` just runs the tests.

If you have a better suggestion I'm happy to rename them.

Reply:
How about something like `build-and-test` and `test`? Maybe too verbose 😅

Reply:
`test` _should_ (in my opinion) be the one that runs everything, as this is what people are used to running.

We could just ditch the `tests` script and people can run `npx jest` or whatever?

Reply:
I've decided to leave this as is, as it'll allow us to run the tests alone with yarn without knowing the test suite, and also run the tests with additional options such as `--coverage` if in the future we want to.